content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of the webhooks, to
+manage their configurations.
+
+## Before you begin
+
+* Install Istio with [DNS certificates configured](/docs/tasks/security/dns-cert) and
+[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) set to `true`.
+
+* Install [`jq`](https://stedolan.github.io/jq/) for JSON parsing.
+
+## Check webhook certificates
+
+To display the DNS names in the webhook certificates of Galley and the sidecar injector, we get the secret
+from Kubernetes, parse it, decode it, and view the text output with the following commands:
+
+{{< text bash >}}
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+{{< /text >}}
+
+The output from the above commands should include the DNS names of Galley and the sidecar injector, respectively:
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+{{< /text >}}
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+{{< /text >}}
+
+## Enable webhook configurations
+
+1.  To generate the `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` configuration files, run the following
+command. The [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml) YAML file
+contains an example DNS certificate configuration. Learn more in [the certificate guide](/docs/tasks/security/dns-cert).
+The following command uses the following configurations:
+
+    * The default Istio configuration.
+    * The DNS certificate configuration in `values-istio-dns-cert.yaml`.
+
+You can find more information in the `helm template` command [documentation](https://helm.sh/docs/helm/#helm-template).
+
+    {{< text bash >}}
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
```

Comment:
any equivalent istioctl cmd to generate the manifest?  I'm asking since the overall doc is depromoting helm.

Reply:
Done: changed to use istioctl to generate the manifest.